colab/tflite_text_classification.ipynb

```diff
@@ -1,500 +1,327 @@
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run using IREE\n",
+    "\n",
+    "Overview:\n",
+    "\n",
+    "1.   Import the TFLite model to TOSA MLIR \n",
+    "2.   Compile the TOSA MLIR into an IREE flatbuffer and VM module\n",
+    "3.   Run the VM module through IREE's runtime to test the text classification function\n",
+    "\n",
+    "Both runtimes should generate the same output.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Convert TFLite model to TOSA MLIR with IREE's import tool.\n",
+    "IREE_TFLITE_TOOL = iree_tflite.get_tool('iree-import-tflite')\n",
+    "!{IREE_TFLITE_TOOL} {ARTIFACTS_DIR}/text_classification.tflite -o={ARTIFACTS_DIR}/text_classification.mlir\n",
+    "\n",
+    "with open(ARTIFACTS_DIR.joinpath(\"text_classification.mlir\")) as mlir_file:\n",
+    "  tosa_mlir = mlir_file.read()\n",
+    "\n",
+    "# The generated .mlir file could now be saved and used outside of Python, with\n",
+    "# IREE native tools or in apps, etc."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# The model contains very large constants, so recompile a truncated version to print.\n",
+    "!{IREE_TFLITE_TOOL} {ARTIFACTS_DIR}/text_classification.tflite -o={ARTIFACTS_DIR}/text_classification_truncated.mlir -mlir-elide-elementsattrs-if-larger=50\n",
+    "\n",
+    "with open(ARTIFACTS_DIR.joinpath(\"text_classification_truncated.mlir\")) as truncated_mlir_file:\n",
+    "  truncated_tosa_mlir = truncated_mlir_file.read()\n",
+    "  print(truncated_tosa_mlir, end='')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Compile the TOSA MLIR into a VM module.\n",
+    "compiled_flatbuffer = compile_str(tosa_mlir, input_type=\"tosa\", target_backends=[\"vmvx\"])\n",
+    "vm_module = iree_rt.VmModule.from_flatbuffer(compiled_flatbuffer)\n",
+    "\n",
+    "# Register the module with a runtime context.\n",
+    "config = iree_rt.Config(\"vmvx\")\n",
+    "ctx = iree_rt.SystemContext(config=config)\n",
+    "ctx.add_vm_module(vm_module)\n",
+    "invoke_text_classification = ctx.modules.module[\"main\"]\n",
+    "\n",
+    "def classify_text_iree(text):\n",
+    "  result = invoke_text_classification(tokenize_input(text)).to_host()[0]\n",
+    "  print(result)\n",
+    "  interpret_output(result)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"Invoking text classification with IREE\\n\")\n",
+    "print(positive_text)\n",
+    "classify_text_iree(positive_text)\n",
+    "print()\n",
+    "print(negative_text)\n",
+    "classify_text_iree(negative_text)"
+   ]
+  }
+ ],
+ "metadata": {
+  "colab": {
+   "collapsed_sections": [
+    "dweksnWs1eZb"
+   ],
```

Comment:
This was collapsing the "Load the TFLite model" section by default and doesn't appear to be anymore. I'm fine with it either way, though I have a slight preference for collapsing the frontend-specific sections to focus on the IREE details. I think updating this to `6Uyx8n_vdqJC` will match the previous behavior

[google/iree `main` branch](https://colab.research.google.com/github/google/iree/blob/main/colab/tflite_text_classification.ipynb#scrollTo=dweksnWs1eZb): 
![image](https://user-images.githubusercontent.com/4010439/155185832-efdb110d-06da-4f98-aae5-19bd32125d7e.png)

[jpienaar/iree `lite` branch](https://colab.research.google.com/github/jpienaar/iree/blob/lite/colab/tflite_text_classification.ipynb#scrollTo=6Uyx8n_vdqJC)
![image](https://user-images.githubusercontent.com/4010439/155186101-3e035ecb-458b-4fe0-905d-fc2669bc2af3.png)